plugins/active_directory_logs.yaml

```diff
@@ -38,13 +38,23 @@ template: |
       start_at: '{{ .start_at }}'
       attributes:
         log_type: "active_directory.general"
+      operators:
+        - id: save_log_record_original
+          type: copy
+          from: body
+          to: attributes["log.record.original"]
```

Comment:
This change should be opt in, please wrap in a boolean option. (For all modified plugins)

When we update Bindplane sources, we can enable this option behind the scenes.